### DISCUSSION TITLE
Livewire temporaryUrl() not showing on product (Shared Hosting)

### DISCUSSION THREAD

**Discussion by MarkLester10** [livewire/livewire#3699] [1. Help]
I have this issue that my image preview files are not displaying on production but working locally,
I deploy my project on Hostinger, where the public folder is changed to public_html

The src attribute of the image is in the attached images;

I find the temporary files storing in storage/app but when I use the temporaryUrl() it seems like it's not fetching the image.

I've also tried working on livewire config file and publishing the assets, but not all of them seems to work.

![capture_20210911134340753](https://user-images.githubusercontent.com/42936075/132937745-ae54416e-f53f-43c8-abd9-e087ef21a148.jpg)
![capture_20210911134301422](https://user-images.githubusercontent.com/42936075/132937746-89f1cd7a-8070-4df9-b1b8-3149c2db86f2.jpg)
![capture_20210911133915647](https://user-images.githubusercontent.com/42936075/132937748-ba4c3547-15e0-4c8b-8956-f741dc6427f6.jpg)

**Comment by Sabawoon-Khan** [comment]
what was your solution?
